docs/dev/Troubleshooting/Troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+#Troubleshooting
+
+## Resolving Issues with Installing Updates
+
+Sometimes, when you install an update of Acuminator, the code analysis and, therefore, diagnostics may stop working. 
+To resolve this issue, you can try the following approaches:
+ - Check and remove remaining Acuminator files from the Visual Studio folder
+ - Clear MEF cache using an extension
+ 
+Before you try any of the approaches, you need to remove the Acuminator extension.
+ 
+### Removing Acuminator Files
+In some cases, the Acuminator files from the previous version may remain in the Local folder of Visual Studio.
+To remove these Acuminator files, do the following:
+1. Locate the Visual Studio folder. The location of the folder may depend on the installation settings. An example location is the following where <User_Name> is the name of your user in Windows:
+   `C:\Users\<User_Name>\AppData\Local\Microsoft\VisualStudio`
+   If you do not see the AppData folder in the File Explorer, you need to enable showing of hidden folders in the Control Panel.
```

Comment:
I thought such instruction can be omitted because it is not related to Acuminator and usually people know how to do that.
For exampe, I did it multiple times and probably would have found the setting by myself if you didn't tell me where to look.

Reply:
People usually don't change these settings multiple times. They do it at most once when they configure machine, some people don't even know that they have hidden folders since it requires some knowledge about the operation system. 
When I configured this setting for my machine last time I had to Google where to find this setting. I don't mind if we just provide a link to one of such guides like this one:
https://www.technipages.com/show-hidden-files-windows